src/rebar_prv_escriptize.erl

```diff
@@ -80,11 +80,16 @@ do(State) ->
                     ?PRV_ERROR({bad_name, Name})
             end
     end,
-    AppInfo1 = rebar_hooks:run_all_hooks(Cwd, pre, ?PROVIDER, Providers, AppInfo0, State),
-    ?INFO("Building escript...", []),
-    Res = escriptize(State, AppInfo1),
-    rebar_hooks:run_all_hooks(Cwd, post, ?PROVIDER, Providers, AppInfo1, State),
-    Res.
+    case AppInfo0 of
```

Comment:
I am trying to understand if this is meant to go upstream too.

Reply:
I see this already proposed (and merged!) upstream. Thanks.

https://github.com/erlang/rebar3/pull/2027/commits/93e300b7657d34111f11257e23aebb629ef21438

Reply:
This is already landed in upstream master. I backported it here, until the next rebar3 release is being used.